### PR TITLE
Grafana dashboard for Nginx metrics

### PR DESCRIPTION
**Pull request by Zempashi**
Fixes: #143

### CODE REVIEW

File: roles/kube_prometheus/vars/main.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+kube_prometheus_dashboards:
+  - kube-prometheus-calico2-dashboard
+  - kube-prometheus-calico-dashboard
+  - kube-prometheus-elasticsearch-dashboard
+  - kube-prometheus-etcd-dashboard
+  #- kube-prometheus-node-exporter-dashboard
+  - kube-prometheus-prometheus-dashboard
```
Comment (NicolasT) [inline roles/kube_prometheus/vars/main.yml]: If the `nginx-ingress` dashboard is not listed here, it will not be deployed :)

**Review by NicolasT — CHANGES_REQUESTED** [review]
I believe the dashboard needs to be added to the list of deployed dashboards in the role

File: roles/kube_prometheus/vars/main.yml (hunk 1 above)
Comment (Zempashi) [reply roles/kube_prometheus/vars/main.yml]: Yes. We also might want to put this in `defaults/main.yml` to be overriden if needed (a dashboard that get problem to be deployed for example...)

Comment (NicolasT) [reply roles/kube_prometheus/vars/main.yml]: Agree, at some later point :)

File: third-party.yaml [hunk 2]
```diff
@@ -52,6 +52,11 @@ modules:
   # the Calico license
   # See https://github.com/projectcalico/calico/pull/1167
   license: Apache-2.0
+- path: vendor/kubernetes/ingress-nginx
+  source:
+    type: url
+    base: https://raw.githubusercontent.com/kubernetes/ingress-nginx/master
+  licence: Aparche-2.0
```
Comment (NicolasT) [inline third-party.yaml]: `Apache-2.0`

File: roles/kube_prometheus/hack/fix-dashboard.py [hunk 3]
```diff
@@ -12,9 +12,14 @@
 def is_dashboard(dashboard):
     return all([
         'annotations' in dashboard,
-        'rows' in dashboard,
+        'rows' in dashboard or 'panels' in dashboard,
     ])
 
+def _iter_panels(dashboard):
+    for row in dashboard.get('rows') or [dashboard]:
```
Comment (NicolasT) [inline roles/kube_prometheus/hack/fix-dashboard.py:19]: I'd write `dashboard.get('rows', [dashboard])`, but fine :)

**Review by NicolasT — CHANGES_REQUESTED** [review]
Please sync with target branch and fix the license typo.

**Comment by NicolasT** [comment]
I'm on it.

**Review by NicolasT — APPROVED** [review]

**Pull request merged**